src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java

```diff
@@ -109,26 +106,36 @@ public void transition_creates_atask_with_default_due_date() {
             .body("[0].groupId", is("TCW"));
     }
 
+
     @Test
-    public void transition_creates_atask_with_due_date() {
+    public void transition_creates_a_task_with_due_date() {
+
+        ZonedDateTime dueDate = ZonedDateTime.now().plusDays(2);
+        Map<String, DmnValue<?>> processVariables = mockProcessVariables(
+            dueDate.toString(),
+            "Provide Respondent Evidence",
+            "provideRespondentEvidence",
+            "external"
+        );
+
         given()
             .relaxedHTTPSValidation()
             .header(SERVICE_AUTHORIZATION, serviceAuthorizationToken)
             .contentType(APPLICATION_JSON_VALUE)
-            .body(requestRespondentEvidenceTaskRequest(caseId)).log().body()
+            .body(new SendMessageRequest("createTaskMessage", processVariables)).log().body()
             .baseUri(testUrl)
-            .basePath("/tasks")
+            .basePath("/workflow/message")
             .when()
             .post()
             .then()
-            .statusCode(HttpStatus.CREATED_201);
+            .statusCode(HttpStatus.NO_CONTENT_204);
 
         Object taskId = given()
             .header(SERVICE_AUTHORIZATION, serviceAuthorizationToken)
             .contentType(APPLICATION_JSON_VALUE)
             .baseUri(camundaUrl)
             .basePath("/task")
-            .param("processVariables", "caseId_eq_" + caseId)
+            .param("processVariables", "caseReference_eq_" + caseReference)
```

Comment:
according to the task-configuration this should be caseId
The service is overdue a cleanup and refactoring but it's defined here:
https://github.com/hmcts/wa-task-configuration-api/blob/4fecb40f7e82a20281e6c13bf238d9e195b2f99c/src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/ccdmapping/ConfigureTaskService.java#L22

Reply:
ok, I changed it because the contract message from CCD says caseReference. But yeah, I change it now.